code/__HELPERS/time.dm

```diff
@@ -11,25 +11,6 @@
 
 #define TICKS2DS(T) ((T) TICKS)
 
-#define TimeOfGame (get_game_time())
-#define TimeOfTick (world.tick_usage*0.01*world.tick_lag)
```

Comment:
These defines were only used for this, were formatted wrong, and quite honestly fucking hurt me